### DISCUSSION TITLE
Will nhost V2 offer same features as V1?

### DISCUSSION THREAD

**Discussion by Svarto** [nhost/nhost#89] [Q&A] [answered]
It is a general question, and more specifically for my use case is the image transformation on-the-fly that V1 offers - will this be supported in V2 and any indication of timeline for implementation?

**Comment by elitan** [comment, marked as answer]
Yes, we do have the same image transformation. Just as previously, just add `w`, `h`, `r`, `b`, or `q` as query parameters to any image served via Nhost.

We'll add better support for image transformation in our JavaScript SDK. Tracked here: https://github.com/nhost/hasura-storage-js/issues/1
